snippets/queryStringToObject.md

```diff
@@ -0,0 +1,30 @@
+---
+title: queryStringToObject
+tags: object,intermediate
+---
+
+Returns a key-value pairs of object of query params of given url.
+
+- Use `String.prototype.split()` to split url .
+- Return the object of key value pair of query params of given url.
+
+```js
+
+function queryStringToObject(url){
+    var query = url.split('?');
+    if(query[1]){
+	    var queryParams = {};
+	    var val = '';
+	    query = query[1].split('&');
+	    query.forEach((e) => {
+	        val = e.split('=');
```

Comment:
You could improve this to consider the case where the variable itself contains `=`, like `http://foo.bar?one=one&two=two=two`.

Reply:
`=` in params are not a valid character and are usually serialized/escaped.